packages/core/src/errors-list.ts

```diff
@@ -68,6 +68,11 @@ export const ERROR_RANGES: {
     max: 899,
     title: "Status errors",
   },
+  DEPLOY: {
```

Comment:
I added a deploy section, for errors that are expected in the deploy task outside of execution.